client/src/components/FAQPanel/FAQPanel.yaml

```diff
@@ -0,0 +1,20 @@
+question1_q:
```

Comment:
o wait what's the difference from this file to common_questions.yaml?

Reply:
This is the one I made for the FAQPanel storybook

Reply:
ohh👍